app/code/Magento/Store/Model/StoreManager.php

```diff
@@ -242,7 +267,10 @@ public function reinitStores()
     }
 
     /**
-     * {@inheritdoc}
+     * Retrieve default store for default group and website
```

Comment:
Please revert the comment and just remove curly braces `{}`. Please refer to the guide https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html#inheritdoc